### PR TITLE
prep for 4.1.7

### PR DESCRIPTION
**Pull request by costaconrado**
This PR updates the CHANGELOGs for 4.1.7 release.

### CODE REVIEW

**Review by osoukup — COMMENTED** [review]
I would remove the **Unreleased** section completely as it is not really part of this release. Even though the version 4.1.7 does not contain it.

**Review by osoukup — APPROVED** [review]
Actually I probably did not understand the aim of this PR. If it is to update the master changelog, then it is all OK.

**Comment by costaconrado** [comment]
@osoukup the intentions of this PR is just to make sure master's changelog / openapi is up-to-date.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +19,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Cannot modify CVE of existing flaws (OSIDB-3102)
 - Jira metadata collector is not deleting metadata on failure (OSIDB-3219)
+
+## [4.1.7] - 2024-08-15
```
Comment (jobselko) [inline docs/CHANGELOG.md:23]: outdated date

Comment (osoukup) [reply docs/CHANGELOG.md:23]: It is rather outdated PR. It should have already be merged but it was not. The release happened on that date

**Comment by osoukup** [comment]
Closing. The master is OK now so I will do it clean

**Pull request closed without merging**